main.sql

```diff
@@ -305,10 +316,17 @@ CREATE TABLE mag.PaperUrls(
     PaperId BIGINT,
     SourceType SMALLINT,
     SourceUrl TEXT,
-    LanguageCode VARCHAR(10)
+    LanguageCode VARCHAR(10),
+    UrlForLandingPage TEXT,
+    UrlForPdf TEXT,
+    HostType TEXT,
+    Version TEXT,
+    License TEXT,
+    RepositoryInstitution TEXT,
+    OaiPmhId TEXT
   );
 
-COPY mag.PaperUrls(PaperId, SourceType, SourceUrl, LanguageCode) FROM '/home/input/mag/PaperUrls.txt' WITH CSV delimiter E'\t'   QUOTE E'\b'  null as ''; 
+\COPY mag.PaperUrls(PaperId, SourceType, SourceUrl, LanguageCode, UrlForLandingPage, UrlForPdf, HostType, Version, License, RepositoryInstitution, OaiPmhId) FROM '../input/export/mag/PaperUrls.txt' delimiter E'\t' null as '';
```

Comment:
This is the backslash-tab problem covered in issues doc.
`WITH CSV delimiter E'\t'` doesn't work, `delimiter E'\t'` does.